test_data/cloud-init/clear/openstack/latest/user_data

```diff
@@ -58,10 +58,10 @@ write_files:
     content: |
         #!/bin/bash
 
-        mount -t virtio_fs virtiofs /mnt -o rootmode=040000,user_id=0,group_id=0,dax
+        mount -t virtiofs myfs /mnt
         bash -c "echo 0000:00:05.0 > /sys/bus/pci/devices/0000\:00\:05.0/driver/unbind"
         bash -c "echo 1af4 1041 > /sys/bus/pci/drivers/vfio-pci/new_id"
         bash -c "echo 0000:00:06.0 > /sys/bus/pci/devices/0000\:00\:06.0/driver/unbind"
         bash -c "echo 1af4 1041 > /sys/bus/pci/drivers/vfio-pci/new_id"
 
-        /mnt/cloud-hypervisor --kernel /mnt/vmlinux --cmdline "console=hvc0 reboot=k panic=1 nomodules i8042.noaux i8042.nomux i8042.nopnp i8042.dumbkbd root=/dev/vda2 VFIOTAG" --disk path=/mnt/clear-31310-cloudguest.img path=/mnt/cloudinit.img --cpus 1 --memory size=512M --rng --device path=/sys/bus/pci/devices/0000:00:05.0/ path=/sys/bus/pci/devices/0000:00:06.0/
```

Comment:
Did you intent to remove "--rng"? That will make the boot-up slower as the SSH key generation will take a long long time and it might even lead to the test timing out.